frontend/src/cards/storybook/PopulationCard.stories.tsx

```diff
@@ -18,7 +19,10 @@ export default {
 } as Meta;
 
 const Template: Story<PopulationCardProps> = (args) => (
-  <PopulationCard {...args} />
+  <PopulationCard
+    {...args}
+    jumpToData={() => console.log("Jump to Missing Data")}
```

Comment:
Do we want this console.log in here?

Reply:
I did that since in the storybook i don't think the components relate to one another... so there was no way to show it "jumping down". I honestly haven't used storybook at all and probably dig into  it a bit; i know a lot of people like it.